deploy/DEB/postinst

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+#
+# coding=utf-8
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Source debconf library.
+. /usr/share/debconf/confmodule
+
+MADLIB_VERSION="1.15-dev"
+MADLIB_INSTALL_PATH="InstallPathNotFound"
+
+# Fetching configuration from debconf
+db_get madlib/installpath
+MADLIB_INSTALL_PATH=$RET
+
+# Remove existing soft links
```

Comment:
It would be nice to move the post-install/post-remove bash commands to its own function and make it common between deb and rpm.

Reply:
It seems this is not as trivial as it seems. CPackDeb takes a few extra files (`postinst, postrm` etc.) but not any arbitrary file. We would have to put the common file into a different folder such as `/usr/share/madlib`, since we have to access this file after madlib is removed.